torch/fx/graph_module.py

```diff
@@ -15,7 +15,6 @@
 from pathlib import Path
 import os
 import warnings
```

Comment:
Please clean up all the various line changes, spacing, and commented out testing code in the next iteration of this PR. The noise makes it harder to see the substantive changes. Thanks :)